Lib/test/test_asyncio/test_locks.py

```diff
@@ -305,6 +305,12 @@ def test_context_manager(self):
 
         self.loop.run_until_complete(f())
 
+    def test_lock_loop_not_running(self):
+        loop = asyncio.new_event_loop()
+        loop.stop()
```

Comment:
The `loop.stop()` isn't needed here, the event loop is only running with a coroutine function (you can test this by trying to call `asyncio.get_running_loop()` outside of a coroutine). So, even without the `loop.stop()`, it should still trigger the `DeprecationWarning`.

```suggestion
```